cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitor.java

```diff
@@ -183,36 +191,25 @@ public LoadMonitorState state(OperationProgress operationProgress) {
     double minMonitoredPartitionsPercentage = _defaultModelCompletenessRequirements.minMonitoredPartitionsPercentage();
 
     // Get the window to monitored partitions percentage mapping.
-    Map<Long, Double> windowToMonitoredPercentage =
-        _metricCompletenessChecker.monitoredPercentages(clusterModelGeneration(),
-                                                        clusterAndGeneration.cluster(),
-                                                        totalNumPartitions);
+    SortedMap<Long, Float> partitionCoverageByWindows =
```

Comment:
Reminder for the future name change: we removed the term `coverage` from our previous patch.